### PR TITLE
feat(libs): print library information in json rendering

### PR DESCRIPTION
**Pull request by mixth**
## Description
- Show library information in json when `--js-lib` and `--json` flag
- Change json key from `result` to `vulnerabilities` and its empty state to be `[]` instead of string

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Related Issue

#30 

## Screenshots (if appropriate):
Screen Shot 2019-10-20 at 5 26 27 AM

## Checklist:

- [ ] I have updated the documentation (if required).
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] I added a picture of a cute animal cause it's fun

### CODE REVIEW

**Comment by rajkumaar23** [comment]
@mixth a quick typo. under the 'libraries' array, the version field's value has a typo. `avaliable`

File: src/RenderJson.js [hunk 1]
```diff
@@ -57,4 +56,24 @@ module.exports = class RenderJson {
...
+    if (
+      jsLibrariesResult &&
+      jsLibrariesResult.details &&
+      jsLibrariesResult.details.items &&
+      jsLibrariesResult.details.items.length > 0
+    ) {
+      return jsLibrariesResult.details.items.map(jsLib => {
+        return {
+          name: jsLib.name,
+          version: jsLib.version || '(version not avaliable)'
```
Comment (lirantal) [inline src/RenderJson.js]:
```suggestion
          version: jsLib.version || '(version not available)'
```

Comment (lirantal) [reply src/RenderJson.js]: as pointed out at https://github.com/lirantal/is-website-vulnerable/pull/31#issuecomment-546688517 let's resolve this first before merging @mixth

Comment (mixth) [reply src/RenderJson.js]: Done :) Thanks @rajkumaar23! I do fix them both in `json` and `console`.

Comment (lirantal) [reply src/RenderJson.js]: awesome work you two 👏

**Comment by lirantal** [comment]
oh my, I forgot to release this as a major version 😳
apologies in advance!

**Pull request merged**